.evergreen.yml

```diff
@@ -69,12 +70,31 @@ functions:
           WORKDIR: ${workdir}
           GOROOT: ${go_root}
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
-          MCLI_ORG_ID: ${mcli_org_id}
-          MCLI_PRIVATE_API_KEY: ${mcli_private_api_key}
-          MCLI_PROJECT_ID: ${mcli_project_id}
-          MCLI_PUBLIC_API_KEY: ${mcli_public_api_key}
+          MCLI_ORG_ID: ${atlas_org_id}
+          MCLI_PROJECT_ID: ${atlas_project_id}
+          MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+          MCLI_SERVICE: ${atlas_service}
           <<: *go_options
         command: make e2e-test-iam
+  "cloud manager e2e":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          WORKDIR: ${workdir}
+          GOROOT: ${go_root}
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_ORG_ID: ${cloud_manager_org_id}
+          MCLI_PROJECT_ID: ${cloud_manager_project_id}
+          MCLI_PRIVATE_API_KEY: ${cloud_manager_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${cloud_manager_public_api_key}
+          MCLI_SERVICE: ${cloud_manager_service}
+          <<: *go_options
+        command: make e2e-test-cloud
```

Comment:
[q] any reason not to just do `make e2e-test-iam`? probably we can rename the old iam to atlas iam, and this to cloud manager iam

Reply:
My idea was that the command` make e2e-test-cloud` would contain also the e2e/cloud manager folder such as

```
.PHONY: e2e-test-cloud
e2e-test-cloud: build ## Run IAM E2E tests
	@echo "==> Running Cloud Manger E2E tests..."
	# the target assumes the MCLI-* environment variables are exported
	go test -v -p 1 -parallel 1 -tags=e2e ./e2e/iam...
	go test -v -p 1 -parallel 1 -tags=e2e ./e2e/cloud_manager...
```
I think it is handy to have an e2e command for cloud manager which contains also iam. But I think that we can live without it.